lib/navigation.php

```diff
@@ -277,11 +277,24 @@ function gutenberg_migrate_menu_to_navigation_post( $new_name, $new_theme, $old_
 	}
 	remove_filter( 'option_stylesheet', $get_old_theme_stylesheet );
 
-	update_option( 'fse_navigation_areas', $area_mapping );
+	update_option( 'wp_navigation_areas', $area_mapping );
 }
 
 add_action( 'switch_theme', 'gutenberg_migrate_menu_to_navigation_post', 200, 3 );
 
+/**
+ * Retrieves navigation areas.
+ *
+ * @return array Navigation areas.
+ */
+function gutenberg_get_navigation_areas_menus() {
+	$areas = get_option( 'wp_navigation_areas', array() );
+	if ( ! $areas ) {
+		$areas = get_option( 'fse_navigation_areas', array() );
```

Comment:
Once `fse_navigation_areas` has been accessed _once_ should we set the values to `wp_navigation_areas` and then delete the option `fse_navigation_areas`? That way we don't have legacy values hanging around on a given install.

Reply:
It could, but what would this achieve? It's extra complexity that won't lead to removing this `if`.

Reply:
All good points. 

I was thinking what happens if I used `register_navigation_areas()` to re-register nav areas to be an empty array ( I don't know _why_ I'd do this, but it could happen)? In this case I would expect to receive an empty array from `gutenberg_get_navigation_areas_menus()`. But with this implementation we would actually receive the legacy areas stored in `fse_navigation_areas()` indefinitely.